scaleapi/__init__.py

```diff
@@ -918,3 +920,234 @@ def create_training_task(
 
         training_task_data = self.api.post_request(endpoint, body=kwargs)
         return TrainingTask(training_task_data, self)
+
+    def list_teammates(self) -> List[Teammate]:
+        """Returns all teammates.
+        Refer to Teams API Reference:
+        https://docs.scale.com/reference/teams-list
+        Returns:
+            List[Teammate]
+        """
+        endpoint = "teams"
+        teammate_list = self.api.get_request(endpoint)
+        return [Teammate(teammate, self) for teammate in teammate_list]
+
+    def invite_teammates(self, emails: List[str], role: TeammateRole) -> List[Teammate]:
+        """Invites a list of emails to your team.
+
+        Args:
+            emails (List[str]):
+                emails to invite
+            role (TeammateRole):
+                role to invite
+        Returns:
+            List[Teammate]
+        """
+        endpoint = "teams/invite"
+        payload = {
+            "emails": emails,
+            "team_role": role.value,
+        }
+        invited_teammates = self.api.post_request(endpoint, payload)
+        return [Teammate(teammate, self) for teammate in invited_teammates]
+
+    def update_teammates_role(
+        self, emails: List[str], role: TeammateRole
+    ) -> List[Teammate]:
+        """Updates role of teammates by email
+
+        Args:
+            emails (List[str]):
+                emails to update
+            role (TeammateRole):
+                new role
+        Returns:
+            List[Teammate]
+        """
+        endpoint = "teams/set_role"
+        payload = {
+            "emails": emails,
+            "team_role": role.value,
+        }
+        updated_teammates = self.api.post_request(endpoint, payload)
+        return [Teammate(teammate, self) for teammate in updated_teammates]
+
+    def list_studio_assignments(self) -> Dict[str, StudioLabelerAssignment]:
+        """Returns a dictionary where the keys are user emails and the
+        values are projects the user is assigned to.
+
+        Returns:
+            Dict[StudioLabelerAssignment]
+        """
+        endpoint = "studio/assignments"
+        assignments = self.api.get_request(endpoint)
+        return {
+            email: StudioLabelerAssignment(assigned_projects, email, self)
+            for (email, assigned_projects) in assignments.items()
+        }
```

Comment:
nit: we can improve code readability here by making iteration earlier. 

List comprehension is pythonic but having it embedded in a dict then use it in return can be harder to understand. wdyt?

Reply:
Sounds good, yeah would improve readability!!